server.js

```diff
@@ -1,20 +1,126 @@
 const express = require("express");
 const cors = require("cors");
-
+var bodyParser = require('body-parser');
+const { check, validationResult } = require('express-validator');
 const app = express();
-
+const port =process.env.PORT|| 3002;
 app.use(express.json());
 app.use(cors());
+app.use(bodyParser.urlencoded({ extended: true })); 
 
 //Use this array as your (in-memory) data store.
 const bookings = require("./bookings.json");
 
-app.get("/", function (request, response) {
-  response.send("Hotel booking server.  Ask for /bookings, etc.");
+app.get('/', function(request, response) {
+  response.sendFile(__dirname + '/index.html');
 });
+// app.get("/", function (request, response) {
+//   response.send("Hotel booking server.  Ask for /bookings, etc.");
+// });
 
 // TODO add your routes and helper functions here
+app.get('/bookings', function(request, response) {
+  
+  response.json(bookings)
+
+});
+//create new booking
+app.post('/bookings',[check('email').isEmail(),],  (req, res) => {
+  const errors = validationResult(req)
+  if (!errors.isEmpty()) {
+    return res.status(422).json({ errors: errors.array() })}
+  let booking={
+    id:bookings.length+1,
```

Comment:
There is a problem with setting the ID as the length of messages.
For example, if you have 5 bookings with IDs 1,2,3,4,5 and you delete booking 4, then add another message, this new message will have an ID of 5. This will give you 2 messages with an ID of 5, and IDs have to be unique.